nifi-nar-bundles/nifi-sql-reporting-bundle/nifi-sql-reporting-tasks/src/main/java/org/apache/nifi/reporting/sql/connectionstatuspredictions/ConnectionStatusPredictionsEnumerator.java

```diff
@@ -48,7 +48,7 @@ public Object current() {
 
     @Override
     public boolean moveNext() {
-        currentRow = null;
+        //currentRow = null;
```

Comment:
should we remove the line completely?

Reply:
I commented out while I was debugging the unit tests. I'd rather restore it just because it's been there so long.